src/main/java/ch/njol/skript/expressions/ExprPrefix.java

```diff
@@ -0,0 +1,66 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.util.common.AnyPrefixed;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Prefix")
+@Description({"The prefix of a team, a member of a team, a player, or anything that can have a prefix.",
+	"Please note that getting or changing a player's permission-plugin prefix requires Vault "
+		+ "and a compatible permissions plugin.",
+	"Getting or changing a team prefix is supported by default."
+})
+@Examples({
+	"on chat:",
+	"\tcancel event",
+	"\tbroadcast \"%player's prefix%%player's display name%: %message%\" to the player's world",
+	"",
+	"set the player's prefix to \"[&lt;red&gt;Admin&lt;reset&gt;] \"",
+	"",
+	"set {team}'s prefix to \"&lt;blue&gt;Blue Team&lt;reset&gt; \""
+})
+@Since("2.0")
+@RequiredPlugins({"Vault", "a chat plugin that supports Vault"})
+public class ExprPrefix extends SimplePropertyExpression<AnyPrefixed, String> {
+
+	static {
+		register(ExprPrefix.class, String.class, "[chat|team] prefix", "any-prefixed");
```

Comment:
I don't think chat or team being allowed for everything is good. `team prefix of player` could return their chat prefix via luckperms or something. And addons registering an any-prefixed just makes this worse.

Reply:
Yeah this is true, I was just trying to follow the original (I don't like it having `chat` in at all because it's got nothing to do with `chat`)

Reply:
Honestly I think it's fine to remove it entirely